src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -74,10 +74,15 @@ const handler: Processor<string, void, string> = async (job: Job<string>) => {
     return;
   }
 
-  // The transaction may be errored if it is manually retried.
-  // For example, the developer retried all failed transactions during an RPC outage.
-  // An errored queued transaction (resendCount = 0) is safe to retry: the transaction wasn't sent to RPC.
-  if (transaction.status === "errored" && resendCount === 0) {
+  // If the transaction is errored and has not yet been sent,
+  // reset it to a QueuedTransaction to try again.
+  // No transaction hashes means the transaction is not in-flight.
+  if (
+    transaction.status === "errored" &&
```

Comment:
Fixes a wrong assumption where resendCount == 0 means the tx was never sent. ResendCount = 0 just means the tx was never "resent". It may have been sent once.